src/razor/src/extension.ts

```diff
@@ -53,6 +53,7 @@ import { RazorFormatNewFileHandler } from './formatNewFile/razorFormatNewFileHan
 import { InlayHintHandler } from './inlayHint/inlayHintHandler';
 import { InlayHintResolveHandler } from './inlayHint/inlayHintResolveHandler';
 import { getComponentPaths } from '../../lsptoolshost/builtInComponents';
+import { BlazorDebugConfigurationProvider } from './blazorDebug/blazorDebugConfigurationProvider';
```

Comment:
This looks like a copy of line 9.

Reply:
@dibarbet, perhaps we don't need this particular cherry-pick.

Reply:
@ryzngard @phil-allen-msft I'm not seeing the fix in the prerelease branch, so as far as I know we need it
https://github.com/dotnet/vscode-csharp/blob/prerelease/src/razor/src/extension.ts

Reply:
I'll find out why the cherry pick didn't work